### PR TITLE
CLDR-15924 items not picked up in pr2297

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15924

Mike, please look these over. I tried to get all the items, except that I wanted to make sure we wanted the prefix on formal, even for non-long items.

- [ ] This PR completes the ticket.

### CODE REVIEW

**Review by AEApple — APPROVED** [review]

**Comment by AEApple** [comment]
Did you mean to add Mike as a reviewer?

File: common/main/si.xml [hunk 1]
```diff
@@ -9589,12 +9589,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
...
 	<personNames>
 		<nameOrderLocales order="givenFirst">und</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ko si te vi yue zh</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko si ta te vi yue zh</nameOrderLocales>
 		<foreignSpaceReplacement xml:space="preserve"> </foreignSpaceReplacement>
 		<initialPattern type="initial">{0}.</initialPattern>
 		<initialPattern type="initialSequence">{0} {1}</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```
Comment (mimckenna) [inline common/main/si.xml:9597]: This change to add `{prefix}` should be down at lines 9650-9652 in `<personName order="surnameFirst" length="long" usage="referring" formality="formal">`

File: common/main/ta.xml [hunk 2]
```diff
@@ -11105,12 +11105,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
...
 	<personNames>
 		<nameOrderLocales order="givenFirst">und</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ko ta te vi yue zh</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko si ta te vi yue zh</nameOrderLocales>
 		<foreignSpaceReplacement xml:space="preserve"> </foreignSpaceReplacement>
 		<initialPattern type="initial">↑↑↑</initialPattern>
 		<initialPattern type="initialSequence">↑↑↑</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```
Comment (mimckenna) [inline common/main/ta.xml:11113]: This change to add `{prefix}` should be in lines 11166-11168 for `<personName order="surnameFirst" length="long" usage="referring" formality="formal"`

File: common/main/si.xml (hunk 1 above)
Comment (macchiati) [reply common/main/si.xml]: I just retracted this change, and added the {prefix} further down

File: common/main/ta.xml (hunk 2 above)
Comment (macchiati) [reply common/main/ta.xml]: I just retracted this change, and added the {prefix} further down

**Comment by macchiati** [comment]
I also added the @TECHPREVIEW in the dtd.

common/dtd/ldml.dtd
Added to all the elements

common/dtd/ldmlSupplemental.dtd
minor change from running GenerateDtd to ensure that the changes worked

tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartDtdDelta.java
added signs to the charts to show the tech preview status. (cleaned up the code a bit to do that)

tools/cldr-code/src/main/java/org/unicode/cldr/util/DtdData.java
added the new @TECHPREVIEW, and made it work.
Note: slight refactoring to add getElementThrowingIfNull to simplify logic

**Comment by macchiati** [comment]
> Did you mean to add Mike as a reviewer?
I have to do it manually.

**Comment by macchiati** [comment]
I need a review to merge.

**Review by richgillam — APPROVED** [review]
Rubber stamp.

**Pull request merged**